src/main/kotlin/com/quipalup/katydid/logentry/domain/LogEntryError.kt

```diff
@@ -0,0 +1,5 @@
+package com.quipalup.katydid.logentry.domain
+
+sealed class LogEntryError {
```

Comment:
```suggestion
sealed class CreateLogEntryError {
```
We will use this sealed class to document all possible errors in "create log entry" use case.